docs/content/components/cta-strip.mdx

```diff
@@ -58,9 +64,33 @@ code_snippets:
         <div class="usa-section usa-section--dark">
           <div class="grid-container">
             <ul class="nci-cta-strip">
-              <li><a href="#!" class="usa-button">Programs</a></li>
-              <li><a href="#!" class="usa-button">Resources</a></li>
-              <li><a href="#!" class="usa-button">Success Stories</a></li>
+              <li><a href="http://www.cancer.gov" class="usa-button">Programs</a></li>
+              <li><a href="http://www.cancer.gov" class="usa-button">Resources</a></li>
+              <li><a href="http://www.cancer.gov" class="usa-button">Success Stories</a></li>
+            </ul>
+          </div>
+        </div>
+    - title: NCI CTA Strip with External Links
```

Comment:
This is probably a good opportunity to port all of these docs site examples to use the twig template instead of copy/pasting the HTML here